tests/test_options.py

```diff
@@ -387,6 +387,36 @@ def test_opt_table_font_raises():
     assert "Either `font=` or `stack=` must be provided." in exc_info.value.args[0]
 
 
+def test_opt_row_striping():
+
+    gt_tbl_0 = GT(exibble)
+    gt_tbl_1 = GT(exibble).opt_row_striping()
+    gt_tbl_2 = GT(exibble).opt_row_striping().opt_row_striping(row_striping=False)
+
+    assert gt_tbl_0._options.row_striping_include_table_body.value == False
+    assert gt_tbl_1._options.row_striping_include_table_body.value == True
+    assert gt_tbl_2._options.row_striping_include_table_body.value == False
+
+
+def test_tab_options_striping():
+
+    gt_tbl_opt = GT(exibble, id="test").opt_row_striping()
+    gt_tbl_tab_options = GT(exibble, id="test").tab_options(row_striping_include_table_body=True)
+
+    assert gt_tbl_opt.as_raw_html() == gt_tbl_tab_options.as_raw_html()
```

Comment:
Can you test this further upstream? If rendered html strings are the very last step, is there an earlier step that indicates these two approaches do the same thing (e.g. checking the options values directly, etc..)?

Reply:
I'll check the options values directly here (comparing the two approaches).